pkg/customMetrics/customMetrics.go

```diff
@@ -0,0 +1,105 @@
+package customMetrics
```

Comment:
I vote to rename this package metrics.

Reply:
The default package provided by SDK for metrics collection is also named as `metrics`.  Just for the differentiation, this is named customMetrics. If that's fine,  shall I revert back to metrics?

Reply:
That is fine with me.